src/styles/withStyles.js

```diff
@@ -42,7 +42,9 @@ const withStyles = styleSheet => BaseComponent => {
               ].join('\n'),
             );
 
-            acc[key] = `${renderedClasses[key]} ${classesProp[key]}`;
+            if (classesProp[key] !== undefined) {
```

Comment:
Do we really want that logic? Isn't the previous warning enough? That's some wasted CPU cycles in production.